readthedocs/proxito/views/hosting.py

```diff
@@ -77,7 +77,10 @@ def _resolve_resources(self):
             project = unresolved_url.project
             version = unresolved_url.version
             filename = unresolved_url.filename
-            build = version.builds.last()
+            build = version.builds.filter(
```

Comment:
```suggestion
            # This query uses a particular index: ``builds_build_version_id_state_date_success_12dfb214_idx``.
            # Otherwise, if the index is not used, the query gets too slow.
            build = version.builds.filter(
```